src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPrintMembers.cs

```diff
@@ -245,7 +245,7 @@ static bool isPrintable(Symbol m)
                     return false;
                 }
 
-                if (m.Kind is SymbolKind.Field)
+                if (m.Kind is SymbolKind.Field && m is not TupleErrorFieldSymbol)
```

Comment:
Could you please clarify what scenario is impacted by this change?

Reply:
`record struct ValueTuple<T1, T2>(T1 Item1, T2 Item2)` would hit an assertion during emit without this.